ui/app/AppLayouts/Profile/views/AdvancedView.qml

```diff
@@ -38,6 +38,7 @@ SettingsContentBase {
 
     property bool isFleetSelectionEnabled
     property bool isBrowserEnabled: true
+    property bool minimizeOnCloseOptionVisible
```

Comment:
Maybe `minimizeOnCloseOptionAvailable` would be more accurate?

edit: I see that in AppMain you call it `minimizeOnCloseOptionAvailable` too, so IMo we could keep that name at all times 😄